src/main/java/UserCommunicator.kt

```diff
@@ -0,0 +1,37 @@
+import java.io.BufferedReader
+import java.io.InputStreamReader
+
+object UserCommunicator {
+    fun askUserConsent(reasonStr: String): Boolean {
+        println("Dear user, $reasonStr, \n Do you want to continue? (Y/n)")
+        var inputString = readLine()
```

Comment:
```suggestion
        val inputString = readLine()
```
Could be val since it is used only once and not changed later.